bindings/ruby/tests/CMakeLists.txt

```diff
@@ -4,4 +4,19 @@
 # This software may be modified and distributed under the terms of the
 # BSD-3-Clause license. See the accompanying LICENSE file for details.
 
-# TODO
+macro(add_ruby_unit_test rubyscript)
+  find_program(RUBY_EXECUTABLE
```

Comment:
`RUBY_EXECUTABLE` should be already defined by the call to `find_package(Ruby)` (see similar comment in the python bindings).